### PR TITLE
Vue package

### PR DESCRIPTION
**Pull request by haverchuck**
*Description of changes:*
First pass at refactoring for the aws-amplify-vue package.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,88 +39,80 @@ $ npm start
 
 Check http://localhost:8080/
 
-## AWS Amplify Integration Explained
-
-This sample demostrate some of the AWS Amplify integration with VueJs. Most of the Amplify logics are contained inside `src/amplify` folder.
```
Comment (mlabieniec) [inline README.md:44]: Typo: demonstrates

**Review by mlabieniec — COMMENTED** [review]
Small typo otherwise looks good

File: README.md (hunk 1 above)
Comment (haverchuck) [reply README.md:44]: That block is being deleted.

**Review by mlabieniec — APPROVED** [review]
👍

**Pull request merged**